TravelAgency-service/src/test/java/cz/muni/fi/travelAgency/service/TripServiceTest.java

```diff
@@ -0,0 +1,135 @@
+package cz.muni.fi.travelAgency.service;
+
+import cz.muni.fi.travelAgency.config.ServiceConfiguration;
+import cz.muni.fi.travelAgency.dao.TripDao;
+import cz.muni.fi.travelAgency.entities.Trip;
+import org.mockito.*;
+import org.springframework.test.context.ContextConfiguration;
+import org.springframework.test.context.testng.AbstractTestNGSpringContextTests;
+import org.testng.Assert;
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.Test;
+
+import java.time.LocalDate;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+@ContextConfiguration(classes = ServiceConfiguration.class)
+public class TripServiceTest extends AbstractTestNGSpringContextTests {
```

Comment:
Add tests where the DAO throws and exception and assert that the exception Simona created is thrown from the Service.